src/BinSkim.Driver/ExpandArguments.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.IL
+{
+    public static class ExpandArguments
+    {
+        public static string[] GenerateArguments(
+            string[] args,
+            IFileSystem fileSystem,
+            IEnvironmentVariables environmentVariables)
+        {
+            var expandedArguments = new List<string>();
+
+            foreach (string argument in args)
+            {
+                if (!IsResponseFileArgument(argument))
+                {
+                    expandedArguments.Add(argument);
+                    continue;
+                }
+
+                string responseFile = argument.Trim('"').Substring(1);
```

Comment:
I see what you're saying, I took this from the original implementation. I think you're correct here, though this concerns only a situation when you're escaping characters (Who knows what the original reason was).

in windows CLI: `BinSkim.exe analyze "@example.rsp"` would treat the double quotes as a single parameter `@example.rsp` (quotations are needed because of PowerShell)
in Linux CLI: `./BinSkim analyze "@example.rsp"` or `./BinSkim analyze @example.rsp` are both valid.

It would be an issue if we escape the quotation marks: `./BinSkim analyze \"@example.rsp\"`. At that point it would break this implementation but `./BinSkim analyze @example.rsp\"\"\"\"` wouldn't.

I'll make this a tiny bit more robust, so that one can input an escaped quote.